Dockerfile

```diff
@@ -1,19 +1,26 @@
-FROM machalen/bowtie2
+FROM ubuntu:jammy
 
+USER root
```

Comment:
Is this line necessary?  (I used it previously to get around some bioconda-specific config, but I think it'd be fine to drop it here